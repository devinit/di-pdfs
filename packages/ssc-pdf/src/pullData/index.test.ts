import * as prettyFormat from 'pretty-format';
import {gqlFileContent} from '.';

describe('Pull Data', () => {
    it('read in gql Files', async () => {
        const data = gqlFileContent();
        expect(prettyFormat(data)).toMatchSnapshot();
     });
});
