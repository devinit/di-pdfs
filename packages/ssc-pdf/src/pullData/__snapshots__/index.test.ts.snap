// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pull Data read in gql Files 1`] = `
"Array [
  \\"query Brazil {
  dvptCooperation(country: \\\\\\"brazil\\\\\\") {
    id
    uid
    year
    color
    value
  }
  tchCooperationByRegion{
    region
    color
    year
    value
    uid
  }
  topTchRecipients(country: \\\\\\"brazil\\\\\\") {
    recipient
    value
  }
}
\\",
  \\"query China {
  dvptCooperation(country: \\\\\\"china\\\\\\") {
    id
    uid
    year
    color
    value
  }
}
\\",
  \\"query SouthAfrica {
  dvptCooperation(country: \\\\\\"southAfrica\\\\\\") {
    id
    uid
    year
    color
    value
  }
}
\\",
  \\"query Inda {
  dvptCooperation(country: \\\\\\"inda\\\\\\") {
    id
    uid
    year
    color
    value
  }
}
\\",
]"
`;
